website/src/pages/log-streaming/configure-streaming-for-third-party-tools/index.mdx

```diff
@@ -81,9 +81,7 @@ Download the `connect-distributed.properties` [here](https://github.com/IBM/clou
 - Modify the `plugin.path` in the file contents to point to the directory `connector` we created in the previous step.
 - Now copy the file `connect-distributed.properties` to the `config` folder under the `kafka_2.13-2.5.0` folder.
 
-### Run Kafka connect
-
-Open a terminal. Run the below commands. The [base dir] is the directory under which we created the folder `kafka_splunk_integration`.
+### Run Kafka connectOpen a terminal. Run the below commands. The [base dir] is the directory under which we created the folder `kafka_splunk_integration`.
```

Comment:
This has a markdown formatting error. The heading `Run Kafka Connect` is immediately followed by text instructions. There needs to be an empty line between a markdown heading and following text.